jwst/rscd/rscd_sub.py

```diff
@@ -217,8 +217,8 @@ def get_rscd_parameters(input_model, rscd_model):
         log.warning('No matching row found in RSCD reference table for')
         log.warning('READPATT=%s, SUBARRAY=%s, Row type=EVEN',
                     readpatt, subarray)
-        raise NoTableEntry('No match for READPATT or SUBARRAY in RSCD ref table')
-        return None, None
+        log.warning('RSCD correction will be skipped, only 1 Group need at least 2')
```

Comment:
The last phrase here "only 1 Group need at least 2" is not relevant to this situation. The appropriate reason was already stated in the warning message in line 217. So this message can be trimmed to just "RSCD step will be skipped."

Reply:
it is not rscd_sub that returns NONE but a function in rscd_sub (get_rscd_parameters) that returns NON to rscd_sub. If none is returned from get_rscd_parameters to rscd_sub
then a warning message is sent that the table value could not be found, output.meta.cal_step.rscd is set to skipped and output is returned (output is set to input_model.copy() in the beginning of rscd_sub.

Reply:
I took out that extra (incorrect warning message) in the most recent pr